.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,4 +13,9 @@ Contributors guide: https://github.com/apache/rocketmq-client-nodejs/blob/master
 - [ ] `npm test` passes
 - [ ] tests and/or benchmarks are included
 - [ ] documentation is changed or added
-- [ ] commit message follows [commit guidelines](https://github.com/apache/rocketmq-client-nodejs/blob/master/PULL_REQUEST.md#commit-message-guidelines)
+- [ ] commit message follows [commit guidelines](https://github.com/apache/rocketmq-client-nodejs/blob/master/CONTRIBUTING.md#commit-message-guidelines)
+
+##### Description of change
+
++ any feature?
++ Close #[AN_ISSUE_NUMBER]
```

Comment:
It would be better to add another step to validate the change after description of change